api/config/cdr_config_perf.json

```diff
@@ -39,16 +47,34 @@
     {
       "cdrVersionId": 3,
       "isDefault": true,
+      "accessTier": "registered",
       "name": "Synthetic Dataset v3",
-      "bigqueryProject": "all-of-us-ehr-dev",
+      "bigqueryProject": "fc-aou-cdr-perf",
       "bigqueryDataset": "test_R2019q4r3",
-      "creationTime": "2019-09-30 00:00:00Z",
+      "creationTime": "2021-04-20 00:00:00Z",
       "releaseNumber": 3,
       "numParticipants": 234525,
       "cdrDbName": "synth_r_2019q4_14",
+      "elasticIndexBaseName": "cdr",
```

Comment:
Elastic should be empty or unset, since there's no such index in this environment, also microarray dataset should probably not be set for either